### DISCUSSION TITLE
Quartz 3.6 in order

### DISCUSSION THREAD

**Discussion by GabrielSoalheiro** [quartznet/quartznet#1867] [Q&A]
How to arrange jobs in alphabetical order?

**Comment by lahma** [comment]
`jobs.OrderBy(x => x.Group).ThenBy(x => x.Name)`

**Reply by GabrielSoalheiro** [reply]
But this way here, I'm not implementing it, can you help me![image](https://user-images.githubusercontent.com/51724295/208024348-36a882ff-6208-485d-bbf3-968c6179fa16.jpeg)

**Reply by lahma** [reply]
One job is already in alphabetical order.

**Reply by GabrielSoalheiro** [reply]
Yes, but how would you implement it for more than one job?![image](https://user-images.githubusercontent.com/51724295/208132560-52b5be86-68fa-46e2-8ef7-99549f45f260.jpeg)![image](https://user-images.githubusercontent.com/51724295/208132585-59eea138-5090-4988-ab54-966e7aa84687.jpeg)

**Reply by lahma** [reply]
I don't know this UI, maybe it should be implemented there?

**Reply by GabrielSoalheiro** [reply]
Okay, thank you very much
